javascript/javascript3/week1/myFile.json

```diff
@@ -0,0 +1,23 @@
+{
+  "motorcycles": {
```

Comment:
It is ok but as there are a list of subsequent data, it is better to use an array.
so `"motorcycles":[Honda, Yamaha, Ducati]`.